bugcrowd/bmt-ruby
# Add the AI LLM Methodology

# Description

Updates the submodule including the methodology files to latest, which includes the AI LLM Testing Methodology.

# Testing steps
The methodology key is
1. `ai_llm`

```
bin/console
```

```ruby
> BMT.find('ai_LLM').title
1. Confirm you can see the new title `AI LLM Testing`.
